proposals/xxxx-mesh-nodes.md

```diff
@@ -0,0 +1,329 @@
+| `[NodeIsProgramEntry]` | N | Node can be launched directly from the API in addition to or instead of from an upstream node in the work graph. ([details](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/WorkGraphs.md#shader-function-attributes)) |
+| `[NodeID("nodeName")]` or `[NodeID("nodeName",arrayIndex)]` | N | Name for the node; uses function name if omitted. Optional `uint arrayIndex` overrides the default index of `0`. ([details](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/WorkGraphs.md#shader-function-attributes)) |
+| `[NodeLocalRootArgumentsTableIndex(index)]` | N | `uint index` indicates the record index into the local root arguments table bound when the work graph is used. May be omitted or set to `-1` (equivalent to omitting).  If omitted and a local root signature is used, the runtime will auto-assign the index. ([details](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/WorkGraphs.md#shader-function-attributes)) |
+| `[NodeShareInputOf("nodeName")]` or `[NodeShareInputOf("nodeName",arrayIndex)]` | N | Share the input of the specified NodeID with this node. ([details](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/WorkGraphs.md#shader-function-attributes)) |
+
+#### Function Parameters
+
+##### Input Record
+
+Zero or one input record declaration is supported for the mesh node entry.
+Types supported are the same as for a broadcasting launch node shader.
+The input record takes the place of the `payload` in the mesh shader.
+
+| Input | Description |
+|-|-|
+| `DispatchNodeInputRecord<`*recordType*`>` | read only node input |
+| [`globallycoherent`] `RWDispatchNodeInputRecord<`*recordType*`>` | Shared R/W access to input record across launched shaders. `globallycoherent` required for any cross-group coherency. |
+| *none* | Input record can be omitted when there is no record content. `[NodeDispatchGrid(...)]` is then required to specify a fixed grid size. |
+
+One system value is supported inside the *recordType* for `SV_DispatchGrid`,
+see [here](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/WorkGraphs.md#sv_dispatchgrid)
+for more details, where mesh nodes are just like broadcast launch nodes in
+this regard.
+Any other semantics on fields in the record structure are ignored.
+
+See the work graphs spec
+[Node input declaration](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/WorkGraphs.md#node-input-declaration)
+section under options for broadcasting launch for more detail.
+
+##### Input System Values
+
+For input values outside the input record, mesh nodes support the `broadcasting` launch [node shader system values](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/WorkGraphs.md#node-shader-system-values).
+These are the same [system values supported by mesh shaders](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/MeshShader.md#hlsl-attributes-and-intrinsics).
```

Comment:
I think you wanted this anchor: 
```suggestion
These are the same [system values supported by mesh shaders](https://github.com/microsoft/DirectX-Specs/blob/master/d3d/MeshShader.md#system-value-semantics).
```
I think the intent is to say that the meaning of those supported are the same as they are in mesh shaders since there is one omission in ViewID though it is, somewhat strangely, not listed in the system values section of that spec